pulpcore/tests/functional/api/test_role.py

```diff
@@ -27,3 +33,67 @@ def test_contains_permission_filter(roles_api_client):
         )
 
     assert change_task_present and view_taskschedule_present
+
+
+@pytest.mark.parallel
+def test_for_object_type_filter(roles_api_client, role_factory):
+    """Test for_object_type query parameter."""
+
+    group_href = constants.BASE_PATH + "groups/"
+    prefix = uuid.uuid4()
```

Comment:
```suggestion
    prefix = str(uuid.uuid4())
```
Sorry i keep forgetting that...